src/pages/generic/lazy.md

```diff
@@ -137,33 +137,32 @@ CsvEncoder[Bar]                   // 3
 CsvEncoder[Int :: String :: HNil] // 4 uh oh
 ```
 
-Le compilateur essais de résoudre `CsvEncoder[::[H, T]]` deux fois
+Le compilateur essait de résoudre `CsvEncoder[::[H, T]]` deux fois
 dans cette branche, une fois a la ligne 2 et une fois a la ligne 4.
-Le parametre de type `T` est plus complexe a la ligne 4 qu'a la ligne 2,
-donc le compilateur pense (érronément dans ce cas)
-que la recherche dans cette branche n'aboutiras pas.
-Il change de branche encors et encors,
+Le paramètre de type `T` est plus complexe à la ligne 4 qu'à la ligne 2,
+donc le compilateur pense (de façon erronée dans ce cas)
+que la recherche dans cette branche n'aboutira pas.
+Il change de branche encore et encore,
 il ne trouvera pas de quoi générer la bonne instance.
 
 ### *Lazy*
 
-La divergence d'implicit mettrais un 
-stop au bibliotèque telle que shapeless.
-Heureusement, shapeless fournis un 
+La divergence d'implicit peut marquer un arrêt pour la bibliotèque shapeless.
+Heureusement, shapeless fournit un
 type appeler `Lazy` pour contourner ce problème.
-`Lazy` fait deux chose:
+`Lazy` fait deux choses :
 
- 1. Il supprime la divergence d'implicit a la 
-    compilation en ce protégant des heuristique trop défensive.
+ 1. Il supprime la divergence d'implicit à la
+    compilation en se protégeant des heuristiques trop défensives.
 
 
- 2. il reporte l'evaluation des parametre implicite au runtime,
-    ce qui permet la déduction des implicits récusifs.
+ 2. Il reporte l'évaluation des paramètres implicites au runtime,
+    ce qui permet la déduction des implicits récursifs.
 
-On utilise `Lazy` en le parametrant avec nos parametrs implicits.
+On utilise `Lazy` en le paramétrant avec nos paramètres implicites.
 En règle générale, il est toujours bon de mettre dans `Lazy`
-le parametre de "tête" de toute règle de `HList` ou de `Coproduct`
-ansi que le nimporte quel parametre `Repr` d'un `Generic`:
+dans le parametre de "tête" de toute règle de `HList` ou de `Coproduct`
```

Comment:
le dans ici n'est pas bon

Reply:
Voici [la phrase originale](https://github.com/underscoreio/shapeless-guide/blob/develop/src/pages/generic/lazy.md) :

"As a rule of thumb, it is always a good idea to wrap the "head" parameter of any HList or Coproduct rule and the Repr parameter of any Generic rule in Lazy"

Je propose :

"En règle générale, il est toujours d'englober dans `Lazy` le paramètre de "tête" de toute règle de `HList` ou de `Coproduct` ainsi que n'importe quel paramètre `Repr` d'un `Generic`"